src/pages/experiments/[experimentId]/data-processing/configure-embedding/components/EmbeddingConfig.jsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from 'react';
+import {
+  Collapse, InputNumber, Form, Select, Typography, Tooltip,
+} from 'antd';
+
+import { QuestionCircleOutlined } from '@ant-design/icons';
+
+const { Option } = Select;
+const { Panel } = Collapse;
+const { Text } = Typography;
+
+const MIN_DIST_TEXT = 'This controls how tightly the embedding is allowed compress points together. Larger values ensure embedded points are more evenly distributed, while smaller values allow the algorithm to optimise more accurately with regard to local structure. Expected range: 0.001 to 0.5. Default is 0.1.';
```

Comment:
can you break up this `MIN_DIST_TEXT` into multiple lines -- I am surprised that our linter didn't complain about this

Reply:
Fixed.